test/optimizer/topn_window_elimination.test_slow

```diff
@@ -2,6 +2,8 @@
 # description: Test Top-N Window Elimination Rule
 # group: [optimizer]
 
+require parquet
+
```

Comment:
Can we remove the `require parquet` here? I don't think it's necessary, and then this test can still be run when the Parquet extension hasn't been built